lib/command_runner.dart

```diff
@@ -492,24 +489,26 @@ int _editDistance(String from, String to) {
           if (i == 0) j else if (j == 0) i else 0,
       ],
   ];
+
   for (var i = 1; i < from.length; i++) {
     for (var j = 1; j < to.length; j++) {
-      // Removals from `from`
+      // Removals from `from`,
```

Comment:
These should be ".", not ",". :)

https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences

Reply:
lol whoops, I am typing with my left hand atm lol